### DISCUSSION TITLE
SNES ROM, header, corrupt or missing error

### DISCUSSION THREAD

**Discussion by Atombuster** [sanni/cartreader#724] [Need help]
I got a Kirby superstar to properly read and download to the SD card but every other card I insert such as super Metroid super Mario world space invaders give me the same issue, saying ROM, header corrupt, or missing and I’ve cleaned all the games they’re like perfectly clean. The cartridges are wiped down with isopropyl alcohol. All the pins are nice and shiny. I will include a picture. It’s pretty much this for all of them except for Kirby superstar. For some reason. I’d like some advice please

![image](https://user-images.githubusercontent.com/126848536/222636317-ba9dd2d9-c69d-420e-89b7-86e5ea5d77f6.jpg)

**Comment by Atombuster** [comment]
I’m using hardware version five And I have the snescic module installed

**Reply by Cephiros** [reply]
Reflow solder joints and look for bridges. Best advice, it sounds like something is not right with the hardware. Either that or your carts arent perfectly clean. A little dirt causes alot of problems sometimes.
